PhysicsTools/NanoAOD/python/BToKmumu_cff.py

```diff
@@ -5,7 +5,7 @@
 BToKmumu=cms.EDProducer("BToKmumuProducer",
                         beamSpot=cms.InputTag("offlineBeamSpot"),
                         vertexCollection=cms.InputTag("offlineSlimmedPrimaryVertices"),
-                        muonCollection=cms.InputTag("slimmedMuons"), #NanoAOD muon collection has pT>3 GeV, can go lower here
+                        muonCollection = cms.InputTag("linkedObjects","muons"), # ref slimmedMuons: NanoAOD muon collection has pT>3 GeV, can go lower here
```

Comment:
Thanks for the change
At the moment, the default muon collection in NanoAOD has a pT>3 GeV cut though.
`cut = cms.string("pt > 3 && track.isNonnull && isLooseMuon")`
https://github.com/amartelli/cmssw/blob/69fa35f19a35349709c4cae4a74712f45c115de3/PhysicsTools/NanoAOD/python/muons_cff.py#L54
which makes the MuonMinPt=1 irrelevant here
I would change the default muon collection in NanoAOD as for electrons.